### DISCUSSION TITLE
db.<project-id>.supabase.co ssl cert expired

### DISCUSSION THREAD

**Discussion by samschooler** [supabase/supabase#12401] [Questions]
Looks like when I access my db directly the ssl cert is expired on `Fri, 30 Dec 2022 02:01:54 GMT`.

Repro:
- Access directly my `https://db.<project-id>.supabase.co/` url directly.

Potentially related: https://github.com/supabase/supabase/discussions/12389

**Comment by Dankni95** [comment]
Experiencing this now too.

**Comment by hitsthings** [comment]
Happening to me. For future travelers:

```
echo "" | openssl s_client -starttls postgres -connect db.YOURDB.supabase.co:5432 -showcerts &> cert_output.txt
```

You'll find the certificates expire 2 years after you created the project. Restarting the project and DB do not help. I opened a case with support 3 days ago and heard nothing at all back. This is terrible.

**Comment by ptts** [comment]
**Update 2025-10-15:** Supabase Infra support renewed our SSL certificate. ✅

We're having the same issue...

```console
$ openssl s_client -starttls postgres -connect db.<PROJECT_ID>.supabase.co:5432 </dev/null 2>/dev/null | openssl x509 -noout -enddate
notAfter=Oct  3 08:04:53 2025 GMT
```

Contacted Supabase support (SU-261208) but so far no response.

For some reason, another one of our projects (which was created roughly at the same time) expires in 2030
`notAfter=Jan 22 13:00:07 2030 GMT`
